mautic/mautic
# Add grapesjs to core

| Q                                      | A
| -------------------------------------- | ---
| Branch?                                | features
| Bug fix?                               | no
| New feature?                           | yes
| Deprecations?                          | no
| BC breaks?                             | no
| Automated tests included?              | no
| Related user documentation PR URL      | https://docs.mautic.org/en/builders
| Related developer documentation PR URL | TBC
| Issue(s) addressed                     | N/A

#### Description:

This PR brings the GrapesJS codebase as-is from the external repo, into Core. We will then set up the Git Split as we have done with all the other plugins so that they are all being maintained and versioned in sync with mautic/mautic releases.

See #i-composer-support and #mautic-4 on Slack for background/questions/context on this.

The outstanding PRs will need to be pushed up to mautic/mautic once this PR is merged, and issues will also be transferred.

@nickveenhof I think that we will need to update the composer.json once this PR is merged, also (and maybe for testing so it is not overwritten? If so feel free to let me know or directly add a commit to this PR.

#### Steps to test this PR:
1. Clone Mautic and apply this PR
2. Install Mautic in the usual way
3. Check that the builder is available and working as expected
